wodles/aws/tests/data/fixture_parameters.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+AWS_BASE_PARAMS = [
+    [
+        {"access_key": "AAAAAAAAAAAAAAAAAAAA"},
+        {"secret_key": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"},
+        {"iam_role_arn": ""},
+    ],
+]
+
+
+AWS_INTEGRATION_PARAMS = [
+    AWS_BASE_PARAMS[0] + [
+        {"db_name": ""},
+        {"aws_profile": ""}
+    ],
+]
+
+
+AWS_BUCKET_PARAMS = [
```

Comment:
There is little reason to have a `.py` file just for these two variable. Please, considere to add them to the `conftest.py` and remove this file if so.